client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -142,17 +148,40 @@
             </mat-cell>
         </ng-container>
 
-        <mat-header-row *matHeaderRowDef="getColumnDefinition()"></mat-header-row>
-        <mat-row
-            [ngClass]="selectedRows.indexOf(row) >= 0 ? 'selected' : ''"
-            (click)="selectItem(row, $event)"
-            *matRowDef="let row; columns: getColumnDefinition()"
-            class="lg"
-        >
-        </mat-row>
-    </mat-table>
+                <mat-header-row *matHeaderRowDef="getColumnDefinition()"></mat-header-row>
+                <mat-row
+                    [ngClass]="selectedRows.indexOf(row) >= 0 ? 'selected' : ''"
+                    (click)="selectItem(row, $event)"
+                    *matRowDef="let row; columns: getColumnDefinition()"
+                    class="lg"
+                >
+                </mat-row>
+            </mat-table>
 
-    <mat-paginator class="on-transition-fade" [pageSizeOptions]="pageSize"></mat-paginator>
+            <mat-paginator class="on-transition-fade" [pageSizeOptions]="pageSize"></mat-paginator>
+        </span>
+        <span *ngSwitchCase="'tiles'">
+            <os-grid-layout>
+                <os-block-tile *ngFor="let category of tileCategories" (clicked)="changeToViewWithCategory(category)" [orientation]="'horizontal'" [only]="'title'" [blockType]="'node'" [data]="category" title="{{ category.name | translate }}">
```

Comment:
Please divide `category` and `tileCategory` everywhere. Otherwise this will lead to confusions...